containers/scripts/deploy.sh

```diff
@@ -13,9 +13,6 @@ CONTAINERS=(
     osh-worker
 )
 
-if [ -z "$IS_PODMAN" ]; then
-    PROFILE='--profile=full-dev'
-fi
```

Comment:
Does this really change anything in the CI environment?  I would expect the condition to evaluate to `false` in CI.  I am afraid this could break local development environment for MacOS users.

Reply:
Because macOS is the environment where the condition evaluates to `true` if I remember correctly.

Reply:
This variable was meant to be used for docker. I have removed the profile as well as the use of this variable from rest of the code.

@lzaoral Could you verify this does not break anything on macOS?

Reply:
@kdudka: @siteshwar is correct here, the condition evaluates to true on any platform that uses `docker` because `--profile` feature of `docker-compose` was not supported in `podman-compose` at the time this script was written.  Note that I've never tried `docker` on macOS due to licensing issues (and `podman` is my tool of choice anyway) so the condition would never evaluate to true on my machine.

Reply:
@jperezdealgaba Could you please check whether the changes from this pull request do no break the development environment on macOS with docker (assuming you are still using it)?

Reply:
[osh.log](https://github.com/user-attachments/files/16539350/osh.log)
I have tested this branch and it seems to work as expected.
I have also attacked the logs in case they are needed

Reply:
@jperezdealgaba Thank you for testing it!